lib/riemann/helpers/event.ex

```diff
@@ -60,6 +56,14 @@ defmodule Riemann.Helpers.Event do
         |> Map.put(:attributes, attributes)
       end
 
+      defp machine_hostname do
+        {:ok, hostname} = :inet.gethostname
+        :erlang.list_to_binary(hostname)
+      end
+
+      defp event_host_setting do
+        Application.get_env(:riemann, :event_host)
```

Comment:
Ideally, i'd prefer not to call this every time we build an event (since i believe that `Application.get_env` is a `GenServer.call`).

If you can think of other way to cache it, I'd prefer that, but if not, it might not be a huge deal.